TheJumpCloud/support
# Sa 3140 set new policy functions

## Issues
* [SA-3143](https://jumpcloud.atlassian.net/browse/SA-3143) - Set/New-JCPolicy 

## What does this solve?

Two new functions, Set/New-JCPolicy created to help administrators programmatically set policies from the JumpCloud PowerShell module

## Is there anything particularly tricky?

Yes, some functionality must be tested by hand.

## How should this be tested?

Manual Test Cases are defined in the `Set-JCPolicy.tests.ps1` and `Set-JCPolicy.tests.ps1` files under the skipped 'Manual Test Cases' Context. These tests should be executed manually before approving the PR.

## Screenshots


[SA-3143]: https://jumpcloud.atlassian.net/browse/SA-3143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
